.circleci/config.yml

```diff
@@ -97,7 +119,12 @@ jobs:
     executor: clang6
     environment: { CMAKE_FLAGS: -DSIMDJSON_GOOGLE_BENCHMARKS=ON }
     steps: [ init_clang6, cmake_test_all, cmake_install_test ]
-  
+  # libcpp
+  libcpp-clang9:
+    description: Build and run tests on clang 6 and AVX 2 with a cmake static build and libc++
+    executor: clang9
+    environment: { CMAKE_FLAGS:  -DSIMDJSON_USE_LIBCPP=ON}
```

Comment:
Don't care if you change this in this PR, but:

Extra space here, and missing space at EOL. Best to make this line read like others:

```yaml
    environment: { CMAKE_FLAGS: -DSIMDJSON_USE_LIBCPP=ON }
```

Reply:
Fixed. That was easy.